source/tutorials_wine.rst

```diff
@@ -320,14 +320,14 @@ Paste this code into the file:
    :caption: wine_connection.odahu.yaml
    :name: Wine connection
 
-    kind: Connection
-    id: wine-tutorial
-    spec:
-      type: gcs
-      uri: gs://<paste your bucket address here>/data-tutorial/wine-quality.csv
-      region: <paste region here>
-      keySecret: <paste key secret here>
-      description: Wine dataset
+   kind: Connection
+   id: wine-tutorial
+   spec:
+     type: gcs
+     uri: gs://<paste your bucket address here>/data-tutorial/wine-quality.csv
+     project: <paste project name here>
```

Comment:
Why then we insert in 'region' parameter data from 'project ID' or should I take this information from somewhere else?

Or why this parameter is named 'region'?
![image](https://user-images.githubusercontent.com/49116723/73169118-06eb9b00-4104-11ea-86b6-0e869a78e2e9.png)

Reply:
Different clouds have different names for the parameter. So we choose one name.
You can find the GCP connection documentation here https://docs.odahu.org/ref_connections.html#google-cloud-storage .

Reply:
> Revert the change because there is no project parameter.

Why then we insert in 'region' parameter data from 'project ID' (on GCP) or should I take this information from somewhere else?

Or why this parameter is named 'region'?